prime-router/settings/organizations-local.yml

```diff
@@ -18,10 +18,20 @@
     senders:
       - name: default
         organizationName: waters
-        topic: covid-19-otc
+        topic: covid-19
```

Comment:
I'm moving incoming waters data into our main "topic".  However, there is no waters schema in production yet, so this is only for staging.  Still, we have to think about what rules we want in place.  If we do nothing, the data will flow freely to states.